packages/lib/src/interactions/SelectionTool.tsx

```diff
@@ -109,10 +118,18 @@ function SelectionTool(props: Props) {
       setSelection(undefined);
 
       if (onSelectionEnd && shouldInteract(sourceEvent)) {
-        onSelectionEnd(computeSelection(worldPt));
+        const finalSelection = computeSelection(worldPt);
```

Comment:
```suggestion
        const endSelection = computeSelection(worldPt);
```

Feels odd to call it `final` as there is still the transform left :shrug: